cheatsheets/Transport_Layer_Protection_Cheat_Sheet.md

```diff
@@ -1,411 +1,214 @@
+The [LetsEncrypt](https://letsencrypt.org) CA provides free domain validated SSL certificates, which are trusted by all major browsers. As such, consider whether there are any benefits to purchasing a certificate from a CA.
 
-To avoid the "Which Directory?" problem, a server should provide the user with all required certificates used in a path validation.
+For internal applications, an internal CA can be used. This means that the FQDN of the certificate will not be exposed (either to an external CA, or publicly in certificate transparency lists). However, the certificate will only be trusted by users who have imported and trusted the internal CA certificate that was used to sign them.
 
-### Rule - Be aware of and have a plan for the SHA-1 deprecation plan
+## Use CAA Records to Restrict Which CAs can Issue Certificates
 
-In order to avoid presenting end users with progressive certificate warnings, organizations must proactively address the browser vendor's upcoming SHA-1 deprecation plans. The Google Chrome plan is probably the most specific and aggressive at this point: [Gradually sunsetting SHA-1](http://googleonlinesecurity.blogspot.com/2014/09/gradually-sunsetting-sha-1.html)
+Certification Authority Authorization (CAA) DNS records can be used to define which CAs are permitted to issue certificates for a domain. The records contains a list of CAs, and any CA who is not included in that list should refuse to issue a certificate for the domain. This can help to prevent an attacker from obtaining unauthorised certificates for a domain through a less-reputable CA. Where it is applied to all subdomains, it can also be useful from an administrative perspective by limiting which CAs administrators or developers are able to use, and by preventing them from obtaining unauthorised wildcard certificates.
 
-If your organization has no [SHA256 compatibility issues](https://support.globalsign.com/customer/portal/articles/1499561-sha-256-compatibility) then it may be appropriate to move your site to a SHA256 signed certificate/chain. If there are, or may be, issues - you should ensure that your SHA-1 certificates expire before 1/1/2017.
+## Always Provide All Needed Certificates
 
-## Server Protocol and Cipher Configuration
+In order to validate the authenticity of a certificate, the user's browser must examine the certificate that was used to sign it and compare it to the list of CAs trusted by their system. In many cases the certificate is not directly signed by a trusted CA, but is instead signed by an intermediate CA, which is in turn signed by the root CA.
```

Comment:
*In many cases the certificate is not directly signed by a trusted CA* should be *In many cases the certificate is not directly signed by a root CA*, no?